lightningd/invoice.c

```diff
@@ -267,7 +267,8 @@ static void json_invoice(struct command *cmd,
 static const struct json_command invoice_command = {
     "invoice", json_invoice, "Create an invoice for {msatoshi} with {label} "
 			     "and {description} with optional {expiry} seconds "
-			     "(default 1 hour) and optional {preimage} "
+			     "(default 1 hour) and optional {fallback addresses}"
```

Comment:
The name of the parameter is `fallbacks` (and it's in the lightning-invoice manual page, I checked!).  So perhaps
`optional {fallbacks} addresses (default empty list)`?